test/integrations/destinations/braze/router/data.ts

```diff
@@ -299,13 +327,13 @@ export const data = [
                     JSON: {
                       subscription_groups: [
                         {
-                          external_id: ['user123'],
+                          external_ids: ['user123', 'user12345'],
```

Comment:
I see in API its `external_id` not `external_ids`
ref: https://www.braze.com/docs/api/endpoints/subscription_groups/post_update_user_subscription_group_status/#request-parameters

Reply:
using V2

Reply:
using V2 now